config/seata-config-apollo/src/main/java/io/seata/config/apollo/ApolloConfiguration.java

```diff
@@ -54,17 +54,21 @@
     private static final String APP_ID = "appId";
     private static final String APOLLO_META = "apolloMeta";
     private static final String APOLLO_SECRET = "apolloAccesskeySecret";
+    private static final String APOLLO_CLUSTER = "cluster";
```

Comment:
APOLLO_CLUSTER = "seata"  Is that better?

Reply:
yes